package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon --exec babel-node src/index.js",
-    "data": "nodemon --exec babel-node src/database/test_data/test_data1.js"
+    "data": "nodemon --exec babel-node src/database/test_data/test_data3.js"
   },
```

Comment:
can we have a command for create and drop the database

Reply:
we will with the test